src/transfer.ts

```diff
@@ -74,3 +74,53 @@ export async function ensureFunded(funding: EnsureFundedParams, algod: Algodv2,
 
   return undefined
 }
+
+interface TransferAssetParams extends SendTransactionParams {
+  senderAccount: SendTransactionFrom
+  receiverAccount: Account
+  assetId: number
+  amount: number | bigint
+  transactionParams?: SuggestedParams
+  clawbackFrom?: SendTransactionFrom
+  note?: TransactionNote
+}
+
+export async function transferAsset(
+  { senderAccount, receiverAccount, assetId, amount, transactionParams, clawbackFrom, note, ...sendParams }: TransferAssetParams,
+  algod: Algodv2,
+) {
+  const transaction = algosdk.makeAssetTransferTxnWithSuggestedParamsFromObject({
+    from: getSenderAddress(senderAccount),
+    to: receiverAccount.addr,
+    closeRemainderTo: undefined,
+    revocationTarget: clawbackFrom ? getSenderAddress(clawbackFrom) : undefined,
+    amount: amount,
+    note: encodeTransactionNote(note),
+    assetIndex: assetId,
+    suggestedParams: await getTransactionParams(transactionParams, algod),
+    rekeyTo: undefined,
+  })
+
+  if (!sendParams.skipSending) {
+    if (sendParams.maxFee === undefined) {
+      sendParams.maxFee = (0.02).algos()
```

Comment:
Given we allow peopel to override maxFee I think it's better to not include this because AlgoKit probably shouldn't be opinionated about how much fees people are happy to spend